boto/s3/inventory.py

```diff
@@ -0,0 +1,506 @@
+    </Schedule>
+    <IncludedObjectVersions>All</IncludedObjectVersions>
+    <OptionalFields />
+  </InventoryConfiguration>
+  <IsTruncated>false</IsTruncated>
+  <!-- If ContinuationToken was provided in the request. -->
+  <ContinuationToken>XXX</ContinuationToken>
+  <!-- if IsTruncated == true -->
+  <IsTruncated>true</IsTruncated>
+  <NextContinuationToken>YYY</NextContinuationToken>
+</ListInventoryConfigurationsResult>
+"""
+
+class InventoryFilter(object):
+    """
+    Specifies an inventory filter. The inventory only includes objects
+    that meet the filter's criteria.
+
+    :ivar prefix: The prefix that an object must have to be included
+        in the inventory results.
+    """
+    def __init__(self, prefix=None):
+        self.prefix = prefix
+    def startElement(self, name, attrs, connection):
+        return None
+    def endElement(self, name, value, connection):
+        if name == 'Prefix':
+            self.prefix = value
+    def to_xml(self):
+        s = ''
+        if self.prefix is not None:
+            s = '<Filter>'
+            s += '<Prefix>%s</Prefix>' % self.prefix
+            s += '</Filter>'
+        return s
+
+class InventorySchedule(object):
+    """
+    Specifies the schedule for generating inventory results.
+
+    :ivar frequency: Specifies how frequently inventory results
+        are produced. Valid Values: "Daily" | "Weekly"
+    """
+    def __init__(self, frequency=None):
+        self.frequency = frequency
+    def startElement(self, name, attrs, connection):
+        return None
+    def endElement(self, name, value, connection):
+        if name == 'Frequency':
+            self.frequency = value
+    def to_xml(self):
+        s = ''
+        if self.frequency is not None:
+            s = '<Schedule>'
+            s += '<Frequency>%s</Frequency>' % self.frequency
+            s += '</Schedule>'
+        return s
+
+class KMS(object):
+    """
+    Specifies the use of SSE-KMS to encrypt delivered inventory reports.
+
+    :ivar keyid: Specifies the ID of the AWS Key Management Service
+        (AWS KMS) symmetric customer managed customer master key (CMK)
+        to use for encrypting inventory reports.
+    """
+    def __init__(self, keyid=None):
+        self.keyid = keyid
+    def startElement(self, name, attrs, connection):
+        return None
+    def endElement(self, name, value, connection):
+        if name == 'KeyId':
+            self.keyid = value
+    def to_xml(self):
+        s = ''
+        if self.keyid is not None:
+            s += '<SSE-KMS>'
+            s += '<KeyId>%s</KeyId>' % self.keyid
+            s += '</SSE-KMS>'
+        return s
+
+class InventoryEncryption(object):
+    """
+    Contains the type of server-side encryption used to encrypt
+    the inventory results.
+
+    :ivar kms: Specifies the use of SSE-KMS to encrypt
+        delivered inventory reports.
+
+    :ivar s3: Specifies the use of SSE-S3 to encrypt delivered

```

Comment:
Should this be a boolean? currently you are using None or True. But then lower in to_xml() you are always adding the s3 tag if the value was anything but None. So False would also add the tag. Just wonder if making it a boolean and not having None as a value would help better define it?

Reply:
Ok. Updated.